backend/open_webui/routers/chats.py

```
@@ -1,5 +1,5 @@
-import json
 import logging
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)